distributed/deploy/spec.py

```diff
@@ -99,6 +99,52 @@ async def finished(self):
     def __repr__(self):
         return f"<{type(self).__name__}: status={self.status}>"
 
+    def _repr_html_(self):
```

Comment:
it would be great if there was a kwarg that we could pass a dictionary to which would then add more items to the table. That way for the subclasses of `ProcessInterface` they could bundle up useful information and pass it to the super.

For example in `distributed.deploy.ssh.Process` it would be great to be able to pass things like the PID of the SSH process.

https://github.com/dask/distributed/blob/9569e08f28df3b870048ce7ae8b44961973af6c1/distributed/deploy/ssh.py#L19

Reply:
See `Cluster` for an example of how we do this there 

https://github.com/dask/distributed/blob/9569e08f28df3b870048ce7ae8b44961973af6c1/distributed/deploy/cluster.py#L365